nsv13/code/game/gamemodes/overmap/objectives/scan.dm

```diff
@@ -0,0 +1,43 @@
+/datum/overmap_objective/scan
+	name = "Scan anomalies"
+	desc = "Scan a target anomaly_type"
+	brief = "Locate and scan target anomaly_type"
+	var/anomaly_type
```

Comment:
```suggestion
	var/obj/effect/overmap_anomaly/anomaly_type
```

Reply:
This variable stores a typepath, not an actual object or reference, so it's unspecified.